test/e2e/data/infrastructure-docker/v1.5/bases/cluster-with-kcp.yaml

```diff
@@ -0,0 +1,101 @@
+---
+# DockerCluster object referenced by the Cluster object
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
```

Comment:
All files in this folder should be identical to the ones in the main folder (except that we don't need mp.yaml here)

Reply:
> Create a new v1.4 folder. It should be created based on the main folder and only contain the templates we use in the clusterctl upgrade tests (as of today cluster-template and cluster-template-topology).

(just for reference)